### PR TITLE
Extend information in README

### PR DESCRIPTION
**Pull request by umulmrum**
| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | -
| License       | MIT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,9 @@ License
...
 
 This bundle is licensed under the MIT License. See the complete license under `LICENSE` for details.
+This repository also includes works under other copyright notices. See `THIRD-PARTY-LICENSE` for details.
 
-Reporting an issue or a feature request
+Reporting an Issue or a Feature Request
 ---------------------------------------
 
-TODO
+Please report issues and feature requests at the official [GitHub repository](https://github.com/chameleon-system/chameleon-shop-theme-bundle/issues).
```
Comment (UlrichKu) [inline README.md:25]: All this is Github and official?

Comment (umulmrum) [reply README.md:25]: "official" is meant as opposed to any other sources where the user might have obtained a copy of the repository. I wouldn't rely completely upon the file being read on GitHub.

Comment (UlrichKu) [reply README.md:25]: see (new) wording in https://github.com/chameleon-system/chameleon-base/pull/108

Comment (umulmrum) [reply README.md:25]: This repository is independent of chameleon-system/chameleon-system - I would keep the sentence as is.

**Pull request merged**